Dockerfile.windows

```
@@ -166,37 +175,33 @@ RUN powershell -NoProfile -ExecutionPolicy Bypass -File C:\src\make_lua_lib.ps1
-# Stage 4: output — merge build + piper into final image
+# Stage 4: output — merge build + piper + dcs into final image
+#           Uses nanoserver (minimal) since we only copy files
 # ============================================================
-FROM mcr.microsoft.com/windows/servercore:ltsc2022 AS output
+FROM mcr.microsoft.com/windows/nanoserver:ltsc2022 AS output
+
+# Compiled DLL from build stage
+COPY --from=build C:\\build\\bin\\HoundTTS.dll  C:\\dist\\base\\Mods\\Services\\HoundTTS\\bin\\HoundTTS.dll
+
+# Lua scripts + config from build context
+COPY dcs/Mods/Services/HoundTTS/Scripts/             C:\\dist\\base\\Mods\\Services\\HoundTTS\\Scripts\\
```

Hadolint (2.14.0)

[error] unexpected newline
expecting '\', --checksum, --chmod, --chown, --exclude, --link, a space separated list of file paths, at least one space, only one --from, or other flag

(DL1000)